### PR TITLE
Adds link to Mutalyzer Normalizer on tx overview to visualize official HGVS nomenclature

### LINKED ISSUE #3460: Add genomic reference sequence
It would be useful to add annotation with the genomic (chromosomal) RefSeq on the variant page, especially for copy number variants. An example on correct reporting from the latest ISCN (genome build GRCh38):

NC_000023.11:g.89555676_100352080del

### PR DESCRIPTION
**Pull request by northwestwitch**
In order to fix #3460, I've tried to introduce some code when the variant page loads to interrogate [the service](https://mutalyzer.nl/) and retrieve the official description in real time. Soon enough I've realized that the query takes too much time and loading of variant page gets too slow.

So I've decided to add the link to the service on the HGVS transcripts overview.. Better than nothing!

image

<hr>

image

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR, DN

### CODE REVIEW

File: tests/server/test_links.py [hunk 1]
```diff
@@ -72,6 +73,15 @@ def test_ucsc_link():
     assert link is not None
 
 
+def test_mutalyzer_link(app):
```
Comment (dnil) [inline tests/server/test_links.py:76]: 🥇

**Review by dnil — APPROVED** [review]
Good stuff! Works just fine for exotic SNVs. Will run into issues on the mutalyzer side when going for a little more exotic variants, but that is already towards what the PR is going after: being able to identify those and doing something about them manually.

**Comment by northwestwitch** [comment]
> Good stuff! Works just fine for exotic SNVs. Will run into issues on the mutalyzer side when going for a little more exotic variants, but that is already towards what the PR is going after: being able to identify those and doing something about them manually.

Alternatively one could also use [variantvalidator](https://rest.variantvalidator.org/), which has a much faster Rest API. I'm drafting a PR with  a validation of the ResSeq:HGVSs to the ClinVar add variant page. If this works better we could use it also on the variant's page

**Comment by dnil** [comment]
Looks very good with variantvalidator as wemm, but I don't have much experience with it. mutalyzer is well used, so at least I feel comfortable this way for now, and we can test it out a bit.

**Comment by northwestwitch** [comment]
> Looks very good with variantvalidator as wemm, but I don't have much experience with it. mutalyzer is well used, so at least I feel comfortable this way for now, and we can test it out a bit.

My exact idea. Thanks for the review! :)

**Pull request merged**